app/main.py

```diff
@@ -1 +1,33 @@
-# write your code here
+from __future__ import annotations
+
+
+class Animal:
+    alive = []
```

Comment:
The `alive` list is a class variable that tracks all instances of `Animal`. This could lead to unintended side effects if not managed carefully, especially in a multi-threaded environment. Consider using a different approach to manage the state of instances.